assets/shaders/octree/_coneTrace.glsl

```diff
@@ -75,10 +75,10 @@ vec4 coneTrace(
     Node previousNode = Node(0, vec3(0), 0.0);
     Node previousParentNode;
     int steps = 0;
-    float firstStep = voxelSize;
-
-    distanceAlongCone += firstStep;
-    while (distanceAlongCone < maxDistance && returnColor.a < 1.0) {
+    
+    // Move the cone origin so it doesn't intersect with own voxels
+    vec3 offsetedConeOrigin = coneOrigin + coneDirection * voxelSize * 2;
```

Comment:
This is the same as the firstStep

Reply:
It's similar. In firstStep we take a step within the cone (so the LOD may change) so when we take the first sample we may not be in the most detailed level of the octree. In this case, we ensure the first sample will be of the most detailed level of the octree